### ISSUE TITLE
Ergoemacs doesn't save settings on Mac OS X

### ISSUE THREAD

**Issue by ceridwen** [ergoemacs/ergoemacs-mode#464]
With ErgoEmacs mode 20170509.1202, installed through Melpa, when I change the paste behavior from pasting multiple times to cycling through previous pastes through the ErgoEmacs menu and select "Save Settings for Future Sessions," it returns to the default behavior after closing and starting Emacs again.  This is with Emacs.app 25.2 on Mac OS 10.12.5.  My .emacs in my home directory is writable by my user.

**Comment by mattfidler** [comment]
Hm.  That is dissappointing.  

You could always revert to the old was of saving the settings.

```lisp
(setq ergoemacs-smart-paste nil) ;; Repeat paste.  Paste the same thing multiple times (default)
(setq ergoemacs-smart-paste 'browse-kill-ring) ;; Repeated past browses the kill ring.
(setq ergoemacs-smart-paste 't) ;; Repeated paste, cycles through the last of past items
```

I would like to solve the bugs `ergoemacs-mode` soon.  If you can figure out any of them, I would love some help.

**Comment by ceridwen** [comment]
Yeah, I edited my .emacs to include the smart paste setting.  In my case, adding `'(ergoemacs-smart-paste t)` to custom-set-variables worked.  (This is probably not the recommended method, but I tried some other approaches and couldn't get them to work.)  I wish I could offer more help, but I don't have very much experience with Lisp, and I know next to nothing about Emacs-Lisp.
